tests/test_parse_fasta.py

```diff
@@ -17,7 +17,7 @@ def test_read_file_corectly_parse_fasta():
     """ Tests if FASTA-files are correctly parsed """
 
     script_dir = os.path.dirname(__file__)  # <-- absolute dir the script is in
-    rel_path = "files_for_testing\sequencesfasta_test.sec"
```

Comment:
Using `/` as a path separator in Python is not portable - it won't work on Linux or Mac. Best is to use `Path` from the `pathlib` module and use `/` instead - Python will then translate this to the correct path separator depending on the OS the code is run from.